docs/api-reference/api-reference.md

```diff
@@ -171,13 +171,13 @@ Returns a list of all segments for each datasource enabled in the cluster.
 
 Returns a list of all segments for one or more specific datasources enabled in the cluster.
 
-`GET /druid/coordinator/v1/metadata/segments?includeOvershadowedStatus`
```

Comment:
Are we removing the existing query parameter and adding a new one? Removing it would be a breaking change for clients that use it.

Reply:
Reverted this to includeOvershadowedStatus